src/sentry/integrations/repositories.py

```diff
@@ -13,10 +13,10 @@ def get_repositories(self):
         >>> def get_repositories(self):
         >>>     return self.get_client().get_repositories()
 
-        return [{
-            'name': 'repo_name',
-            'full_name': 'account_name/repo_name',
-        }]
+        return {
+            'name': display_name,
+            'identifier': external_repo_id,
```

Comment:
Can probably just be `id`

Reply:
for GitHub we actually need the full org/user + repo so wondering if `id` doesn't really say enough about what's needed.

Reply:
also we do want to return an array, yeah?